public/app/plugins/panel/heatmap-new/module.tsx

```diff
@@ -169,7 +169,7 @@ export const plugin = new PanelPlugin<PanelOptions, GraphFieldConfig>(HeatmapPan
       .addNumberInput({
         path: 'hideThreshold',
         name: 'Hide cell counts <=',
-        defaultValue: 0.000_000_001, // 1e-9
```

Comment:
not technically needed for Safari, but reduces the changes needed in custom forks to support some older browsers